src/tool/cppxlang/code_writers.h

```diff
@@ -1074,15 +1074,22 @@ namespace xlang
             if constexpr (std::is_base_of_v<Foundation::IUnknown, V>)
             {
                 V result{ nullptr };
-                XLANG_SHIM(Foundation::Collections::IMapView<K, V>)->Lookup(get_abi(key), put_abi(result));
+                com_ptr<xlang_error_info> error_info{
```

Comment:
The Lookup call's result is intentionally ignored. It should probably stay that way in xlang due to the "Try" nature of this helper.

Reply:
Oh, nevermind. You're doing the right thing by capturing and destroying any error object.